app/views/layouts/_header.html.erb

```diff
@@ -1,17 +1,30 @@
 <header id="header" class="navbar navbar-expand-lg navbar-dark fixed-top">
   <div class="container">
-    <%= link_to t("global.header.brand"), "#", id: "logo", class: "navbar-brand" %>
+    <%= link_to t("global.header.brand"), root_url, id: "logo", class: "navbar-brand" %>
```

Comment:
url khác gì path nhỉ

Reply:
+ Path: cung cấp site-root-relative path, tức là đường dẫn tương đối trên trang của mình, cái này dùng nhiều ạ
+ Url: cung cấp đường dẫn tuyệt đối trên trang của mình, thường dùng để dẫn link trang để cho việc xử dụng từ bên ngoài ạ

Reply:
+ Ngoài ra url do nó lấy abosulte path nên nó có cung cấp cả host hay protocol của website mình nữa ạ

Reply:
+ Em có thấy 1 cái recommend là nên dùng url khi redirect ạ, vì nó cung cấp host với protocol của website, gọi là kiểu điều hướng an toàn thôi ạ :v